src/snowflake/sqlalchemy/requirements.py

```diff
@@ -35,6 +35,8 @@ class Requirements(SuiteRequirements):
     - fetch_ties: not supported in snowflake
     - supports_distinct_on: not supported in snowflake
     - time_timezone: not supported in snowflake
+    - array_type: not supported in snowflake
+    - uuid_data_type: not supported in snowflake
```

Comment:
These are new properties (or at least new ones checked in the sqlalchemy test suite) that were added in 2.0. Since 1.4 simply doesn't check them, the addition is backwards-compatible.